diff --git a/node_modules/react-native-onboarding-swiper/src/index.js b/node_modules/react-native-onboarding-swiper/src/index.js
index f3b340d..b55b0c4 100755
--- a/node_modules/react-native-onboarding-swiper/src/index.js
+++ b/node_modules/react-native-onboarding-swiper/src/index.js
@@ -358,31 +358,32 @@ const styles =StyleSheet.create( {
     paddingBottom:30,
     backgroundColor:'#000000',
     // borderRadius:20,
-    shadowColor:
-    Platform.OS == 'android' ? '#D3D3D3' : 'black',
-    shadowOffset: {
-    width:3,
-    height: 2,
-     },
-    shadowOpacity: 3,
-    shadowRadius: 18,
-    elevation: 20,
+    // shadowColor:
+    // Platform.OS == 'android' ? '#D3D3D3' : 'black',
+    // shadowOffset: {
+    // width:3,
+    // height: 2,
+    //  },
+    // shadowOpacity: 3,
+    // shadowRadius: 18,
+    // elevation: 20,
     }
     ,
     customTitleStyle:{
     // textAlign:'center',
-    fontSize:16,
+    fontSize:18,
     color:'#ffffff',
-    marginBottom:5,
-    fontFamily:'Manrope-Bold'
+    marginBottom:10,
+    fontFamily:'SF-Pro-Display-Medium',
+    letterSpacing: 0.7
+    
     },
     customSubtitleStyle:{
-    textAlign:'center',
-    fontSize:12,
-    color:'#757575',
-    fontFamily:'Manrope-Regular'
-
-    
+    fontSize:16,
+    color:'#ffffff',
+    // marginBottom:5,
+    fontFamily:'SF-Pro-Display-Regular',
+    letterSpacing: 0.7
     }
 })
 
